reason-react-native/src/apis/PanResponder.md

```diff
@@ -46,32 +176,109 @@ external config:
                                          =?,
     ~onShouldBlockNativeResponder: (Event.pressEvent, gestureState) => bool=?,
     unit
-  ) =>
-  config =
-  "";
-
-[@bs.module "react-native"] [@bs.scope "PanResponder"]
-external create: config => t = "";
-
-[@bs.get] external panHandlers: t => panHandlers = "";
-
-[@bs.get]
-external onMoveShouldSetResponder: panHandlers => callback(bool) = "";
-[@bs.get]
-external onMoveShouldSetResponderCapture: panHandlers => callback(bool) = "";
-[@bs.get]
-external onStartShouldSetResponder: panHandlers => callback(bool) = "";
-[@bs.get]
-external onStartShouldSetResponderCapture: panHandlers => callback(bool) = "";
-[@bs.get] external onResponderReject: panHandlers => callback(unit) = "";
-[@bs.get] external onResponderGrant: panHandlers => callback(unit) = "";
-[@bs.get] external onResponderRelease: panHandlers => callback(unit) = "";
-[@bs.get] external onResponderMove: panHandlers => callback(unit) = "";
-[@bs.get] external onResponderTerminate: panHandlers => callback(unit) = "";
-[@bs.get]
-external onResponderTerminationRequest: panHandlers => callback(bool) = "";
-[@bs.get] external onResponderStart: panHandlers => callback(unit) = "";
-[@bs.get] external onResponderEnd: panHandlers => callback(unit) = "";
+  )
+```
+
+- `panHandlers` is the abstract type for `panHandlers` of a `PanResponder`,
+  which can be accessed by means of the getter method `panHandlers`
+
+- `callback('a)` is the type of the usual callback function specified for each
+  _Gesture Responder System_ handler defined as
+  ```reason
+  type callback('a) = Event.pressEvent => 'a
+  ```
+  where `'a` is either `bool` or `unit`
 
+## Example
+
+As props spread is not possible with the ReasonML JSX 3 syntax, each _Gesture
+Responder System_ callback needs to be separately specified, with the relevant
+value obtained by means of the associated getter.
+
+While props spread may not be available, copy-paste is probably supported by
+your editor...
+
+```reason
+onMoveShouldSetResponder = panHandlers->PanResponder.onMoveShouldSetResponder
```

Comment:
space around = are probably unnecessary as they will be removed by refmt don't you think?